setup.py

```diff
@@ -40,7 +40,8 @@ def read(*parts):
     'pytest-cov==2.6.*',
     'pytest==4.4.*',
     'pycodestyle==2.5.*',
-    'testfixtures==6.6.*'
+    'testfixtures==6.6.*',
+    'kafka-python'
```

Comment:
Oh, we intentionally left this one out. The Kafka dependency would be a problem for some sites. So if you look at the root directory of this repository, the scripts that import Kafka are one level above the package data, not being included with the final Python package for cylc-xtriggers.

This way they serve more as example for some users. Hope that makes sense. Can you revert this one please @ytturi ?

Reply:
Oh, actually, the `install_requires` needs updating. It's showing `cylc-flow==8.0a0`. That's probably why you got the old version. Can you update it on this PR too, please @ytturi ?

Reply:
Sure!, that was why it didn't match the imports as well 👍 

The kafka-python dependency was only on test-level, so it should not be installed if its not for test purposes. I can remove it as well. I'll set the pytest to run only with the xtriggers then 👍

Reply:
Requirements updated